### ISSUE TITLE
error: The preview feature "filteredRelationCount" is not known.

### ISSUE THREAD

**Issue by Hoffmano** [prisma/prisma#16855]
what is happening here?
```
error: The preview feature "filteredRelationCount" is not known. Expected one of: filterJson, referentialIntegrity, interactiveTransactions, fullTextSearch, fullTextIndex, dataProxy, extendedIndexes, tracing, improvedQueryRaw
workspace-api-1  |   -->  schema.prisma:8
workspace-api-1  |    | 
workspace-api-1  |  7 |   provider = "prisma-client-js"
workspace-api-1  |  8 |   previewFeatures = ["filteredRelationCount"]
workspace-api-1  |    | 
workspace-api-1  | 
workspace-api-1  | Validation Error Count: 1
workspace-api-1  |     at RequestHandler.handleRequestError (/app/node_modules/@prisma/client/runtime/index.js:34883:13)
workspace-api-1  |     at RequestHandler.handleAndLogRequestError (/app/node_modules/@prisma/client/runtime/index.js:34841:12)
workspace-api-1  |     at /app/node_modules/@prisma/client/runtime/index.js:35387:25
workspace-api-1  |     at processTicksAndRejections (node:internal/process/task_queues:96:5)
workspace-api-1  |     at Proxy._executeRequest (/app/node_modules/@prisma/client/runtime/index.js:35955:22) {
workspace-api-1  |   clientVersion: '4.7.1',
workspace-api-1  |   errorCode: undefined
```

_Originally posted by @Hoffmano in https://github.com/prisma/prisma/discussions/16770#discussioncomment-4415233_

**Comment by Hoffmano** [comment]
Link to docs
https://www.prisma.io/docs/concepts/components/preview-features/client-preview-features#preview-features-promoted-to-general-availability:~:text=filteredRelationCount%20since%20release%204.3.0%20%2D%20Submit%20feedback

**Comment by janpio** [comment]
That is highly unexpected @Hoffmano.

What is your `npx prisma -v` output?
When exactly do you get this error message? (Error msg indicates when running some Prisma Client query?)

**Comment by Jolg42** [comment]
Note that it is only available since [4.3.0](https://github.com/prisma/prisma/issues/15069) but it seems you are using `4.7.1` from the logs?
Could you check the version of the Prisma CLI and Prisma Client in your project?

**Comment by JacMarais** [comment]
It works fine for :
prisma                : 2.30.3
@prisma/client        : 4.9.0
Current platform      : windows
Query Engine (Binary) : query-engine b8c35d44de987a9691890b3ddf3e2e7effb9bf20 (at node_modules\@prisma\engines\query-engine-windows.exe)
Migration Engine      : migration-engine-cli b8c35d44de987a9691890b3ddf3e2e7effb9bf20 (at node_modules\@prisma\engines\migration-engine-windows.exe) 
Introspection Engine  : introspection-core b8c35d44de987a9691890b3ddf3e2e7effb9bf20 (at node_modules\@prisma\engines\introspection-engine-windows.exe)
Format Binary         : prisma-fmt b8c35d44de987a9691890b3ddf3e2e7effb9bf20 (at node_modules\@prisma\engines\prisma-fmt-windows.exe)
Default Engines Hash  : b8c35d44de987a9691890b3ddf3e2e7effb9bf20
Studio                : 0.423.0

but if I run the upgrade of prisma from 2.30.3 -> 4.9.0  it breaks.
Unsure if this helps but it should unblock you for now.
